src/BuildScriptGeneratorCli/ServiceProviderBuilder.cs

```diff
@@ -72,6 +72,18 @@ private static LoggingConfiguration BuildNLogConfiguration([CanBeNull] string lo
 
             var aiKey = disableTelemetry ? string.Empty : Environment.GetEnvironmentVariable(
                 LoggingConstants.ApplicationInsightsInstrumentationKeyEnvironmentVariableName);
+            var aiConnectionString = disableTelemetry ? string.Empty : Environment.GetEnvironmentVariable(LoggingConstants.ApplicationInsightsConnectionStringKeyEnvironmentVariableName);
+            if (!string.IsNullOrEmpty(aiConnectionString))
+            {
+                var aiTarget = new ApplicationInsights.NLogTarget.ApplicationInsightsTarget()
+                {
+                    Name = "ai",
+                    InstrumentationKey = aiConnectionString,
```

Comment:
We are able to provide the connection string to this `InstrumentationKey` property?

Reply:
I only see Instrumentation Key in ApplicationInsightsTarget class. Not seeing Connection String.
Not sure if I need to install an updated version from Nuget Package Manager?